tests/unit/test_zigzag_config_file.py

```diff
@@ -101,6 +101,52 @@ def config_missing_project_id_key(tmpdir_factory):
     return config_path
 
 
+@pytest.fixture(scope='session')
+def module_hierarchy_two_nodes(tmpdir_factory):
+    """config with one value in a jinga template"""
+    config = \
+        """
+        {
+            "zigzag": {
+                "project_id": "12345",
+                "test_cycle": "pike",
+                "module_hierarchy": ["{{ NODE_ONE }}", "{{ NODE_TWO }}"],
+                "path_to_test_exec_dir": "{{ FOO }}"
+            }
+        }
+        """  # noqa
+
+    config_path = tmpdir_factory.mktemp('data').join('./conf.json').strpath
+
+    with open(str(config_path), 'w') as f:
+        f.write(config)
+
+    return config_path
+
+
+@pytest.fixture(scope='session')
+def empty_module_hierarchy(tmpdir_factory):
+    """config with one value in a jinga template"""
```

Comment:
Description doesn't match intention of fixture.

Reply:
Updated